Integrations/integration-ATD.yml

```diff
@@ -37,672 +38,670 @@ configuration:
 
-        return {
-            Type: entryTypes.note,
-            Contents: result.multipleResults,
-            ContentsFormat: formats.json,
-            HumanReadable: md,
-            EntryContext: {
-                "ATD.status": result.status, //backward compatability
-                "ATD.Task(val.taskId==obj.taskId)": result.tasks
-                "ATD.Task(val.Id==obj.Id)": result.tasks
-            }
-        };
-    }
+        return_report(uriSuffix, filename, report_type, '', '', threshold)
 
-    function getTaskIds(jobIds){
-        var results = [];
-        var res;
-        for (i = 0; i < jobIds.length; i++) {
-            res = sendRequest('/php/getTaskIdList.php?jobId='+jobIds[i],'GET', getHeaders());
-            results.push(res);
-        }
-        return results;
-    }
+    def job_or_taskId():
+        uriSuffix = ''
+        args = demisto.args()
+        if ( ('jobId' not in args and 'taskId' not in args) or ('jobId' in args and 'taskId' in args) ):
+            return_error('You must specify one (and only one) of the following: jobId, taskId.')
 
-    function getTaskIdsCommand(){
-        var jobIds = stringArrToIntArr(argToList(args.jobId));
-        var results = getTaskIds(jobIds);
-
-        var multipleMd = [];
-        var ec = [];
-        for (i = 0; i < results.length; i++) {
-            multipleMd.push({
-                taskId: results[i].result.taskIdList,
-                jobId: jobIds[i]
-            });
-            ec.push({
-                taskId: results[i].result.taskIdList,
-                jobId: jobIds[i]
-            });
-        }
+        if 'jobId' in args:
+            uriSuffix = 'jobId=' + str(args['jobId'])
+        else:
+            uriSuffix = 'iTaskId=' + str(args['taskId'])
 
-        var md = tblToMd('ATD TaskIds and JobIds List', multipleMd, ["taskId", "jobId"]);
-        return {
-            Type: entryTypes.note,
-            Contents: results,
-            ContentsFormat: formats.json,
-            HumanReadable: md,
-            EntryContext: {
-                "ATD.Task(val.jobId==obj.jobId)": ec
-            }
-        };
-    }
+        return uriSuffix;
 
-    function prettifyFileUploadRes(fileUploadRes){
-        var prettyFileUpload = {
-            'Id': fileUploadRes.results[0].taskId,
-            'taskId': fileUploadRes.results[0].taskId,
-            'jobId': fileUploadRes.subId,
-            'messageId': fileUploadRes.fileId,
-            'url': fileUploadRes.results[0].url,
-            'srcIp': fileUploadRes.results[0].srcIp,
-            'destIp': fileUploadRes.results[0].destIp,
-            'MD5': fileUploadRes.results[0].md5,
-            'SHA1': fileUploadRes.results[0].sha1,
-            'SHA256': fileUploadRes.results[0].sha256,
-        }
-        return prettyFileUpload;
-    }
+    def detonate(submitType ,sample, timeout, report_type, threshold, fileName):
+        sys.exit(0)
```

Comment:
This doesn't make much sense..

Reply:
no it does not. it will be handled in another PR